docs/docs/cmd/spo/list/list-sensitivitylabel-ensure.mdx

```diff
@@ -0,0 +1,61 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# spo list sensitivitylabel ensure
+
+Applies a default sensitivity label to the specified document library
+
+## Usage
+
+```sh
+m365 spo list sensitivitylabel ensure [options]
+```
+
+## Options
+
+```md defintion-list
+`-u, --webUrl <webUrl>`
+: The URL of the site where the library is located.
+
+`-n, --name <name>`
+: The name of the label.
+
+`-t, --listTitle [listTitle]`
+: The title of the library on which to apply the label. Specify either `listTitle`, `listId`, or `listUrl` but not multiple.
+
+`-l, --listId [listId]`
+: The ID of the library on which to apply the label. Specify either `listTitle`, `listId`, or `listUrl` but not multiple.
+
+`--listUrl [listUrl]`
+: Server- or web-relative URL of the library on which to apply the label. Specify either `listTitle`, `listId`, or `listUrl` but not multiple.
+```
+
+<Global />
+
+## Remarks
+
+!!! attention
+    This command is based on an API that is currently in preview and is subject to change once the API reaches general availability.
```

Comment:
This needs to be updated to a ::: styled docusaurus admonition. Please refer to the guide by Milan.